lib/matplotlib/figure.py

```diff
@@ -1249,6 +1249,103 @@ def colorbar(
         cax.figure.stale = True
         return cb
 
+    def colorbars(self, mappable, shape=(-1, -1), fraction_per_row=0.15, pad=0.05,
```

Comment:
I see this is just a placeholder, but `colorbars` seems too close to `colorbar`. Could this be `multivariate_colorbars`?

Reply:
Agreed. 
`multivariate_colorbars` would definitely work, or alternatively `multivar_colorbars` if we want it a bit shorter.

'variate' is quite a long word, and it has made sense to me to shorten it to 'var' in some cases.
However, looking at it now I think it is more important that we are consistent (which I have not really been so far, with functions such as `ensure_multivariate_norm` and classes like `MultivarColormap`).